### PR TITLE
Resized ward pins

### PR DESCRIPTION
**Pull request by dartharaknus**
Fixes #445

### CODE REVIEW

File: src/components/Match/VisionMap.jsx [hunk 1]
```diff
@@ -20,23 +20,27 @@ const style = (width, iconSize, ward) => {
...
-      width={iconSize}
-      height={iconSize}
+      style={style(width, wardSize, log.entered)}
+      width={wardSize}
+      height={wardSize}
       xmlns="http://www.w3.org/2000/svg"
     >
       <g>
-        <title>Observer</title>
+        <title>{log.type === 'observer' ? 'Observer' : 'Sentry'}</title>
```
Comment (howardchung) [inline src/components/Match/VisionMap.jsx]: externalize the strings

Comment (dartharaknus) [inline src/components/Match/VisionMap.jsx]: Done.

**Comment by howardchung** [comment]
LGTM, some lint issue?

**Comment by dartharaknus** [comment]
I imported 'strings' instead of 'lang'

**Comment by howardchung** [comment]
```
/home/travis/build/odota/ui/src/components/Match/VisionMap.jsx
  26:62  error  Infix operators must be spaced  space-infix-ops
```

**Comment by dartharaknus** [comment]
@howardchung Fixed now.

**Comment by howardchung** [comment]
sync master?  one of the CI checks is failing

**Comment by dartharaknus** [comment]
Rebased on master.

**Pull request merged**